### PR TITLE
Update WISH SX reduction systems tests

### LINKED ISSUE #31009: Increase WISH SX reduction system test coverage
Presently the WISH SX system test does not capture many algorithm calls e.g. CreatePeaksTable. This means that when default behaviour changes we don't realise it will break WISH scripts. This issues is to update the test script based on recent development to the script in 
/mnt/ceph/auxiliary/wish

### PR DESCRIPTION
**Pull request by RichardWaiteSTFC**
**Description of work.**

Want to capture algorithms and basic workflow used in reduction of WISH SX data so have better visibility of when changes will effect them. To this end I have added several new systems tests.

**To test:**

(1) Code review
(2) Jenkins passes

Fixes #31009 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Testing/SystemTests/tests/framework/ISIS_WISHSingleCrystalReduction.py [hunk 1]
```diff
@@ -132,8 +132,147 @@ def validate(self):
...
+        ConvertMDHistoToMatrixWorkspace(InputWorkspace=wsMD_2Dcut, outputWorkspace="wsHisto_2Dcut")
+
+    def validate(self):
+        return "wsHisto_2Dcut", "WISH38237_MD_2Dcut.nxs"
+
+
+class WISHIntegrateSatellitePeaksTest(MantidSystemTest):
+    """
+    This tests the loading and normalisation of data, incl. correction for detector efficiency using vanadium and
+    creation of MD workspace
```
Comment (DannyHindson) [inline Testing/SystemTests/tests/framework/ISIS_WISHSingleCrystalReduction.py:207]: Is this comment applicable? Wasn't sure if it was a copy and paste from the previous test?

**Review by DannyHindson — CHANGES_REQUESTED** [review]
I think strictly speaking any files that are tested in the validate methods (ie reference files) should go in Testing\SystemTests\tests\framework\reference. So WISH38237_MD_2Dcut.nxs should ideally go in there. This isn't always possible if a file is an input to some tests and an output of others though + it's not always strictly adhered to so this is optional.

I've added one other detail point about one of the comments

File: Testing/SystemTests/tests/framework/ISIS_WISHSingleCrystalReduction.py (hunk 1 above)
Comment (RichardWaiteSTFC) [reply Testing/SystemTests/tests/framework/ISIS_WISHSingleCrystalReduction.py:207]: Yes sorry, fixed that now

**Review by DannyHindson — DISMISSED** [review]
I've reviewed the code and all looks good. This will be a useful addition to our automated testing given the fact we've had a couple of WISH related issues in recent months that have only shown up in end user usage of the nightly build.

**Comment by DannyHindson** [comment]
@mantidproject/gatekeepers please could someone merge this PR please? It includes a fix to the ISIS_WISHSingleCrystalReduction system test that is failing regularly in the RHEL7 jobs on Jenkins:

https://github.com/mantidproject/mantid/pull/31772/files#diff-ff9ef4347b17e49998c1420fae79cf81d325d87b58db170e9af134304c4a9dbbR62

This has caused a backlog and the RHEL7 job for this PR is caught up in that so all jobs have passed except that one

**Review by DannyHindson — APPROVED** [review]
Have rerun the system tests locally and the ISIS_WISHSingleCrystalReduction suite now all passes for me following the commit to reinstate the missing file

**Pull request merged**